details/impersonation.md

```diff
@@ -87,7 +89,7 @@ Now that we have configured Test Settings and External Services Mocks, we can tr
 
 It means that we pick the users defined in Settings or Mocks, but also we can enter the username and try to impersonate such user.
 
-[Here](../examples/impersonation/impersonate-user-ui.md) you can find a description how to impersonate a user in ROR Kibana.
+The [step by step instruction](../examples/impersonation/impersonate-user-ui.md) shows on how to impersonate a user using ROR Kibana plugin UI.
```

Comment:
BTW Instructions used like this need to be plural.

"Follow the instructions on how to [impersonate a user using the ROR Kibana plugin UI](...)."